src/components/my-career/VisualizeCareer.jsx

```diff
@@ -1,26 +1,39 @@
-import React, { useContext } from 'react';
+import React, { useState } from 'react';
 
 import PropTypes from 'prop-types';
 import {
-  ActionRow, Button, Hyperlink, Icon, Row, useToggle,
+  ActionRow, Button, Icon, Row, useToggle, TransitionReplace,
 } from '@edx/paragon';
-import { AppContext, ErrorPage } from '@edx/frontend-platform/react';
+import { ErrorPage } from '@edx/frontend-platform/react';
 import { useLearnerSkillLevels } from './data/hooks';
 import { LoadingSpinner } from '../loading-spinner';
 import SpiderChart from './SpiderChart';
 import CategoryCard from './CategoryCard';
+import SearchJobRole from './SearchJobRole';
 
-const VisualizeCareer = ({ jobId }) => {
-  const {
-    enterpriseConfig: {
-      slug,
-    },
-  } = useContext(AppContext);
+const editIcon = () => (
+  <Icon className="fa fa-pencil edit-job-role-icon" screenReaderText="Edit Role" />
+);
 
+const VisualizeCareer = ({ jobId, submitClickHandler }) => {
   const [showInstructions, , , toggleShowInstructions] = useToggle(false);
+  const [isEditable, setIsEditable] = useState(false);
```

Comment:
Same as above, I think it would be better to use `useToggle` here.

Reply:
done